grype/db/v6/severity.go

```diff
@@ -0,0 +1,187 @@
+package v6
+
+import (
+	"fmt"
+	"math"
+	"strings"
+
+	gocvss20 "github.com/pandatix/go-cvss/20"
+	gocvss30 "github.com/pandatix/go-cvss/30"
+	gocvss31 "github.com/pandatix/go-cvss/31"
+	gocvss40 "github.com/pandatix/go-cvss/40"
+
+	"github.com/anchore/grype/grype/vulnerability"
+	"github.com/anchore/grype/internal/log"
+)
+
+func extractSeverities(vuln *VulnerabilityHandle) (vulnerability.Severity, []vulnerability.Cvss, error) {
+	if vuln.BlobValue == nil {
+		return vulnerability.UnknownSeverity, nil, nil
+	}
+	sev := vulnerability.UnknownSeverity
+	if len(vuln.BlobValue.Severities) > 0 {
+		var err error
+		sev, err = extractSeverity(vuln.BlobValue.Severities[0].Value)
```

Comment:
I believe you said this was sorted based on "highest severity first" when building the db? it would probably be good to note that here when someone comes back to this section